<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Training buchen - On Padel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ressources/icons/logo_smalltrans.png</normaloff>ressources/icons/logo_smalltrans.png</iconset>
  </property>
  <widget class="QLineEdit" name="terminName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>121</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Kunde suchen...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="terminPhone">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="txtKundeName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>86</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Kunde suchen*</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtPhone">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>85</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Telefon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ressources/icons/logo-onpadel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxHalle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txtHalle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Halle*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTerminBuchen">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 141, 0);
border-radius: 5px;
color:white;
}
QPushButton:hover{
background-color: rgb(0, 189, 0);
border-radius:5px;
color:white;
}</string>
   </property>
   <property name="text">
    <string>BUCHEN</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>541</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,35);
background-color: rgba(156, 139, 116,120);
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>0</y>
     <width>681</width>
     <height>611</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Fluent Icons</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ressources/images/bg11.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txtTitleAppointment">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Training buchen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>11</month>
     <day>17</day>
    </date>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPaymentStatus">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>--Bitte wählen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>vor Ort - Ausstehend</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bezahlt</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxTrainingTyp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>--Bitte wählen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Einzel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gruppe</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="txtMinut">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>239</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Dauer*</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtPayform">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Zahlung*</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtTyp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Training Typ*</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtDate">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Datum*</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDauer">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>120</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="txtCoach">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Trainer*</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtTime">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>239</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Uhrzeit*</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTrainer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxZeit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearFieldsAppointment">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnClearFieldsAppointment{
	color: rgb(255, 255, 255);
	border-radius:5px;
	background-color: rgb(254, 52, 1);
}
QPushButton#btnClearFieldsAppointment:hover{
	border: 2px solid rgb(255,255,255);

}</string>
   </property>
   <property name="text">
    <string>Felder löschen</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ressources/icons/recycle-bin.png</normaloff>ressources/icons/recycle-bin.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="txtCustomerAnotherInfo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Sonstige Infos</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="customerAnotherInfo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtPrice">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Preis*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="terminePreis">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txtPflichtfelder">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>550</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>*Pflichtfelder</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>txtKundeName</zorder>
  <zorder>txtPhone</zorder>
  <zorder>terminName</zorder>
  <zorder>terminPhone</zorder>
  <zorder>comboBoxHalle</zorder>
  <zorder>txtHalle</zorder>
  <zorder>btnTerminBuchen</zorder>
  <zorder>txtTitleAppointment</zorder>
  <zorder>dateEdit</zorder>
  <zorder>comboBoxPaymentStatus</zorder>
  <zorder>comboBoxTrainingTyp</zorder>
  <zorder>txtMinut</zorder>
  <zorder>txtPayform</zorder>
  <zorder>txtTyp</zorder>
  <zorder>txtDate</zorder>
  <zorder>comboBoxDauer</zorder>
  <zorder>txtCoach</zorder>
  <zorder>txtTime</zorder>
  <zorder>comboBoxTrainer</zorder>
  <zorder>comboBoxZeit</zorder>
  <zorder>btnClearFieldsAppointment</zorder>
  <zorder>txtCustomerAnotherInfo</zorder>
  <zorder>customerAnotherInfo</zorder>
  <zorder>txtPrice</zorder>
  <zorder>terminePreis</zorder>
  <zorder>txtPflichtfelder</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
